logsink/logsink.hpp

```diff
@@ -40,6 +40,23 @@ class FileSink : public google::LogSink
 
   ~FileSink();
 
+  virtual void send(
+      google::LogSeverity severity,
+      const char* full_filename,
+      const char* base_filename,
+      int line,
+      const struct ::tm* tm_time,
+      const char* message,
+      size_t message_len,
+      google::int32 usecs);
+
+  // This method implements the now obsolete LogSink interface without the
+  // `usecs` argument introduced by a glog patch in MESOS-9687.
+  //
+  // At the moment of the writing defining this method is still necessary due
```

Comment:
s/the writing/writing/

Reply:
Thanks! Articles in  germanic languages have always been confusing me, and likely will continue  for a long time ;)